public/modules/base/views/signup.html

```diff
@@ -73,6 +73,21 @@
                     </p>
                 </div>
                 
+                <div class="form-group"
+                     ng-class="{
+                     'has-warning': form.username.$dirty,
+                     'has-success': form.username.$valid && form.username.$dirty && !form.username.$error.mongoose,
```

Comment:
I just copied and pasted the logic from and existing input, perhaps @mravinale can answer to this